tests/Sniffs/TypeHints/data/typeHintDeclarationNoErrors.php

```diff
@@ -631,4 +631,19 @@ public function inheritdocAnnotation($a)
 		return true;
 	}
 
+    /**
+     * @inheritDoc
+     */
+	public function camelizedInheritdoc($a)
+    {
```

Comment:
It looks as spaces here. We use tabs for indenting.

Reply:
Done.